extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -191,4 +197,10 @@ private void addDerivedMetricsRow(Map<String, Object> derivedMetrics, List<Strin
     ));
   }
 
+  private String resourceLabelsKeyMapper(String key)
+  {
+    return Strings.isNullOrEmpty(config.getResourceLabelsPrefix()) ? key :
+        config.getResourceLabelsPrefix() + "." + key;
```

Comment:
It would be more flexible to not hardcode the `.` delimiter in the code and instead have it be part of the prefix itself.  This would allow users with other naming conventions to use those (e.g. delimit by `_`, etc.).

Reply:
I made the `delimiter` as part of `prefix` itself. I was thinking whether it makes sense to have another config for `delimiter`.